plugins/editor/CMakeLists.txt

```
@@ -43,21 +48,26 @@ add_library(sfizz_editor STATIC EXCLUDE_FROM_ALL
     src/editor/GUIHelpers.cpp
     src/editor/GUIPiano.h
     src/editor/GUIPiano.cpp
+    src/editor/DlgAbout.h
+    src/editor/DlgAbout.cpp
     src/editor/ColorHelpers.h
     src/editor/ColorHelpers.cpp
     src/editor/ImageHelpers.h
     src/editor/ImageHelpers.cpp
     src/editor/NativeHelpers.h
     src/editor/NativeHelpers.cpp
     src/editor/layout/main.hpp
+    src/editor/layout/about.hpp
     src/editor/utility/vstgui_after.h
     src/editor/utility/vstgui_before.h
     src/editor/GitBuildId.h
+    ${UI_FILES}
     "${CMAKE_CURRENT_BINARY_DIR}/git-build-id/GitBuildId.c")
 add_library(sfizz::editor ALIAS sfizz_editor)
 target_include_directories(sfizz_editor PUBLIC "src")
 target_link_libraries(sfizz_editor PUBLIC sfizz::messaging sfizz::plugins-common)
 target_link_libraries(sfizz_editor PRIVATE sfizz::vstgui)
+target_compile_definitions(sfizz_editor PRIVATE SFIZZ_VERSION=${CMAKE_PROJECT_VERSION})
```

```suggestion
target_compile_definitions(sfizz_editor PRIVATE "SFIZZ_VERSION=\"${CMAKE_PROJECT_VERSION}\"")
```
change similar to this, and remove the `CMAKE_STR` hack